lib/common.js

```diff
@@ -167,8 +159,7 @@ const getIdxLookups = (fields, lookups) => {
         field === lookup.lookupName &&
         standardLookups.some(
           sLookup =>
```

Comment:
I know this is also from before, but stylistically this could have just been `lookup` rather than `sLookup`. If we needed to differentiate it should be `standardLookup`.